src/InputTypes/Credentials.js

```diff
@@ -12,8 +14,28 @@ class CredentialsTooltipItem {
         this.#data = data
     }
     id = () => String(this.#data.id)
-    labelMedium = (_subtype) => this.#data.username
-    labelSmall = (_subtype) => '•••••••••••••••'
```

Comment:
This is the core of this PR. We show the appropriate label depending on the `origin`.